docs/index.rst

```diff
@@ -3,18 +3,53 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Welcome to imitation's documentation!
-=====================================
+Imitation: Clean implementations of Imitation Learning algorithms
+=================================================================
+
+GitHub repo: http://github.com/HumanCompatibleAI/imitation
+
+
+Main Features
+~~~~~~~~~~~~~
+
+- Built on and compatible with Stable Baselines 3 (SB3).
+- Scripts for training SB3 expert models.
+- Data structures and scripts for loading and storing expert demonstrations.
+- Implementations of Behavioral Cloning, DAgger, GAIL, and AIRL with customizable
```

Comment:
What does customizable SB3 mean?

Reply:
- [ ] Should be something like "training arbitrary SB3 policies and modular reward and discriminator networks when applicable"